tcms/testplans/static/testplans/js/get.js

```diff
@@ -19,26 +19,39 @@ $(document).ready(function() {
                 testCases.not_confirmed.push(data[i]);
             }
         }
-        drawConfirmedTestCases();
+        drawConfirmedTestCases(testCases.confirmed);
     });
 
-    function drawConfirmedTestCases() {
-        var container = $('#confirmed-testcases'),
-            noCasesTemplate = $('#no_test_cases'),
-            testCaseRow = $('#test_case_row')[0],
-            testCaseSummary = $(testCaseRow.content).find('.list-view-pf-main-info a.testcase')[0];
-
-        if (testCases.confirmed.length > 0) {
-            for (var i = 0; i < testCases.confirmed.length; i++) {
-                var testCase = testCases.confirmed[i];
-                testCaseSummary.href = `/case/${testCase.id}/`;
-                testCaseSummary.innerHTML = `TC-${testCase.id}: ${testCase.summary}`;
-                container.append($(testCaseRow).html());
-            }
-        } else {
-            container.append(noCasesTemplate[0].innerHTML);
-        }
+});
 
-        treeViewBind();
+function drawConfirmedTestCases(testCases) {
+    var container = $('#confirmed-testcases'),
+        noCasesTemplate = $('#no_test_cases'),
+        testCaseRow = $('#test_case_row')[0],
+        testCaseSummary = $(testCaseRow.content).find('.list-view-pf-main-info a.testcase')[0];
+
+    if (testCases.length > 0) {
+        for (var i = 0; i < testCases.length; i++) {
+            var testCase = testCases[i];
+            testCaseSummary.innerHTML = getTestCaseRowContent(testCase);
+            container.append($(testCaseRow).html());
+        }
+    } else {
+        container.append(noCasesTemplate[0].innerHTML);
     }
-});
+
+    treeViewBind();
+}
+
+
+function getTestCaseRowContent(testCase) {
+    var rowDetailsTemplate = $('#test_case_row_details')[0],
+        row = $(rowDetailsTemplate.content);
+
+    row.find('.js-test-case-link').html(`TC-${testCase.id}: ${testCase.summary}`);
+    row.find('.js-test-case-priority').html(`${testCase.priority}`);
+    row.find('.js-test-case-category').html(`${testCase.category}`);
+    row.find('.js-test-case-author').html(`${testCase.author}`);
+    row.find('.js-test-case-tester').html(`${testCase.default_tester || '-'}`);
```

Comment:
Drop the `js-` class prefix. This is the legacy naming and we don't really need it here.

Reply:
I disagree that this is legacy naming. I need it to point out that this class is used only in the javascript, and must not be used for styling. Issue #1404 is a recent example.

Reply:
I made a quick search, the classes that start with js-test-case... are used only in this commit. Yes, there are other code that use this prefix "js-" but only for javascript selector, no css is applied with those classes.